invenio_rdm_records/services/github/release.py

```diff
@@ -32,6 +32,39 @@ def metadata(self):
         output = metadata.default_metadata
         output.update(metadata.extra_metadata)
         output.update(metadata.citation_metadata)
+
+        if not output.get("creators"):
```

Comment:
This "pollutes" the release API a bit, I would expect that all the metadata-related operations would be in the metadata class `RDMReleaseMetadata` (or `ZenodoReleaseMetadata`). 

However, that requires that we rework the code and we don't want to do it right now I suppose.